packages/uniforms-antd/src/SubmitField.js

```diff
@@ -8,7 +8,6 @@ const SubmitField = ({className, inputRef, value, ...props}, {uniforms: {error,
         htmlType="submit"
         ref={inputRef}
         type="primary"
-        {...props}
```

Comment:
Why have you removed it? We shouldn't use `filterDOMProps` here as it's not wrapped with `connectField`, but it's still needed.